content/ಅಂಕಿಗಳ ವಿಭಕ್ತಿ.md

```diff
@@ -0,0 +1,97 @@
+
+## ಹದಿಮೂರು
+ಈ ಅಧ್ಯಯನವನ್ನು ಹದಿಮೂರರಿಂದ ಆರಂಭಿಸೋಣ. ಹನ್ನೊಂದು ಹನ್ನೆರಡುಗಳು ಯಾಕಲ್ಲವೆಂದು ಮುಂದೆ ವಿಷದವಾಗುತ್ತದೆ.
+ಹದಿಮೂರರಲ್ಲಿ ಇರುವ `ಹತ್ತು` ಮತ್ತು `ಮೂರು` ಎಂಬ ಎರಡು ಭಾಗಗಳನ್ನು ನೋಡಬೇಕು. ಅವುಗಳ ಮಧ್ಯೆ ಇರುವ ಸಂಬಂಧ ನೋಡಬೇಕು.
+ಹತ್ತರ ಮೂರೇ, ಹತ್ತು ಮತ್ತು ಮೂರೇ, ಹತ್ತರ ನಂತರ ಮೂರೇ ?
+
+### ಹತ್ತು
+ನಾಮಪದಗಳ ನಡುವಿನ ಸಂಬಂಧವನ್ನು ನಾವು ವಿಭಕ್ತಿಯಿಂದ ತೋರಿಸುತ್ತೇವೆ. ಪ್ರತ್ಯಯವನ್ನು ನೋಡಿ ವಿಭಕ್ತಿಯನ್ನು ತಿಳಿದು ಒಂದು ನಾಮ ಇನ್ನೊಂದರ ಮೇಲೆ ಹೇಗೆ ಅವಲಂಬಿಸಿದೆ ಎಂದು ಅರಿಯಬೇಕು. ಹದಿಮೂರರ ಹುಟ್ಟು ಹಳಗನ್ನಡದಲ್ಲಿ ಆದದ್ದು. 10 = ಪತ್ತು. ಪತ್ತು ಶಬ್ದಕ್ಕೆ ವಿಭಕ್ತಿ ಪ್ರತ್ಯಯಗಳನ್ನು ಜೋಡಿಸಿದಾಗ ಕೆಳಗಿನ ಪದಗಳು ಉಂಟಾಗುತ್ತವೆ
+1. ಪತ್ತು
+2. ಪತ್ತಂ (ಹತ್ತನ್ನು)
+3. ಪತ್ತಿಂ (ಹತ್ತನ್ನು ಬಳಸಿ)
+4. ಪತ್ತಕ್ಕೆ (ಹತ್ತಕ್ಕೆ)
+5. ಪತ್ತಿನತ್ತಣಿಂ,ಪತ್ತರದೆಸೆಯಿಂದ, ಪತ್ತಿಂ (ಹತ್ತಿಂದ, ಹತ್ತರಿಂದ)
+6. ಪತ್ತರ (ಹತ್ತರ)
+7. ಪತ್ತೊಳ್ (ಹತ್ತಲ್ಲಿ)
+
+ಇವುಗಳಲ್ಲಿ ಕೆಲವು ಪದಗಳು ಉಪಯೋಗಕ್ಕಿಲ್ಲ. ನಮಗೆ ಅವಷ್ಯವಾಗಿರುವವು 3,5, 6 ನೇ ವಿಭಕ್ತಿಗಳು.
+### ತೃತಿಯೆಯೂ ಪಂಚಮಿಯೂ
+`ಕತ್ತಿಯಿಂ ಕಡಿದಂ`, `ಕತ್ತಿಯಿಂದ ಕಡಿದನು` ಎಂಬಲ್ಲಿ ಕತ್ತಿ ಉಪಕರಣ
+`ಕತ್ತಿಯೊಳ್ ಕಡಿದಂ`, `ಕತ್ತಿಯಲ್ಲಿ ಕಡಿದನು` ಎಂದಾಗಲೂ ಅಷ್ಟೆ, ಕತ್ತಿ ಉಪಕರಣ. ಪ್ರತ್ಯಯ ಏನೆ ಇದ್ದರೂ ವಿಭಕ್ತಿ ಕರಣಕಾರಕದ ತೃತಿಯಾ ವಿಭಕ್ತಿ.
+`ಮರದಿಂ ಫಲ ಬೀೞ್ದುದು`, `ಮರದಿಂದ ಹಣ್ಣು ಬಿದ್ದುದು`
+`ಗುಡ್ಡದಿಂದ ಆಚೆ`, `ಈ ಕೋಲಿಂದ ಉದ್ದ` ಎಂಬಲ್ಲಿ **ಇಂ, ಇಂದ** ಪ್ರತ್ಯಯಗಳಿದ್ದರೂ `ಮರ`, `ಗುಡ್ಡ`, `ಕೋಲು`ಗಳು ಉಪಕರಣಗಳಲ್ಲ ಎಂಬುದು ಸ್ವಯಂ ವೇದ್ಯ. ಮರವಿದ್ದಲ್ಲಿ ಇದ್ದ ಹಣ್ಣು ಈಗ ಇಲ್ಲ. ಇದನ್ನು ಅಪಾದಾನ (ಒಂದು ವಸ್ತುವಿನಿಂದ ದೂರ ಸರಿಯುವುದು) ಕಾರಕ ಎನ್ನುತ್ತಾರೆ, ಇದುವೇ ಪಂಚಮೀ ವಿಭಕ್ತಿಯ . ಸೇಡಿಯಾಪು ಕೃಷ್ಣ ಭಟ್ಟರು ಪಂಚಮೀ ವಿಭಕ್ತಿ ಲೇಖನದಲ್ಲಿ ಈ ವಿಚಾರವನ್ನು ಪ್ರತಿಪಾದಿಸಿದ್ದಾರೆ[1]. `ಇಂ` ಮತ್ತು `ಇಂದ` ಪ್ರತ್ಯಯಗಳು ಮೂಲತಹ ಪಂಚಮೀ ವಿಭಕ್ತಿಗಳು ಎಂಬುದನ್ನು ಸಾಧಿಸಿ ತೋರಿಸಿದ್ದಾರೆ.`ಅಲ್ಲಿಂದ` ಎಂಬ ಶಬ್ದಕ್ಕೆ `ಅಲ್ಲಿ-ಗೆ` ಸಾಪೇಕ್ಷವಾಗಿ ಎಂಬ ಪಂಚಮೀ ಅರ್ಥವೇ ಹೊರತು `ಅಲ್ಲಿ-ಯನ್ನುಪಯೋಗಿಸಿ` ಎಂಬ ತೃತಿಯಾರ್ಥ ಬರುವುದೇ ಇಲ್ಲ.
+
+**ಹತ್ತು ಇದ್ದಲ್ಲಿಂದ**, **ಹತ್ತಕ್ಕೆ ಸಾಪೇಕ್ಷವಾಗಿ** ಎಂಬ ಅರ್ಥಬರಲು ಪಂಚಮೀ ವಿಭಕ್ತಿ ಮತ್ತು ಅದರ ಪ್ರತ್ಯಯವನ್ನು ಪ್ರಯೋಗಿಸಬೇಕೆಂಬುದನ್ನು ನೋಡಿದೆವು. ಪಂಚಮೀ ಪ್ರತ್ಯಯವಾಗಿ `-ದೆಸೆಯಿಂದ`, `-ಅತ್ತಣಿಂ` ಗಳನ್ನು ನಾವು ಎಲ್ಲೂ ಪ್ರಯೋಗಿಸುವುದಿಲ್ಲ, **ಇಂ, ಇಂದ** ಗಳನ್ನೇ ಉಪಯೊಗಿಸುವುದು ಎಂದೂ ನೋಡಿದೆವು.ಆದ್ದರಿಂದ **ಪತ್ತಿಂ** ಶಬ್ದವು **ಪತ್ತರತ್ತಣಿಂ** **ಹತ್ತರದೆಸೆಯಿಂದ** ಕ್ಕೆ ಸಮಾನವಾದ ಪಂಚಮೀ ಪ್ರಯೋಗವೆಂಬುದು ನಿಸ್ಸಂಶಯ.
+
+### ಪತ್ತಿಂ ಮೂರು
+ಹತ್ತರತ್ತಣಿಂದ ಮೂರು, ಹತ್ತರಿಂದ ಆಚೆ ಮೂರು, ಹತ್ತಕ್ಕಿಂತ ಮೂರು (ಜಾಸ್ತಿ) ಎಂಬರ್ಥದಲ್ಲಿ ಹುಟ್ಟಿರುವ ಪದವಿದು.
+ಪತ್ತಿಂ ಮೂರು = ಪತ್ತಿಮ್+ಮೂರು = ಪತ್ತಿಮ್ಮೂರು ರೂಪವೇ ಮೂಲರೂಪವಿರಬಹುದು.
+**ಇದುವೇ ಕನ್ನಡ ಸಂಖ್ಯೆಗಳಲ್ಲಿರುವ ಸ್ವಾರಸ್ಯ**
+
+### ಪದಿಂ, ಪಯಿಂ, ಪನ್
+ಈ ಮೂರು ಪದಗಳೂ **ಪತ್ತಿಂ** ನ ವಿವಿಧ ಅಪಭ್ರಂಶ ರೂಪಗಳು ಎಂಬುದು ಈ ಲೇಖನದ ಮತ.
+ಶಬ್ದಮಣಿದರ್ಪಣದಲ್ಲಿ ಕೇಶಿರಾಜ ಇವುಗಳನ್ನು **ಪತ್ತು**ವಿನ ಆದೇಶರೂಪಗಳು ಎಂದಷ್ಟೇ, ಇದ್ದುದನ್ನು ಗಮನಿಸಿ, ಹೇಳಿದ್ದಾನೆ.ಯಾಕೆ ಎಂಬುದನ್ನು ವಿಶ್ಲೇಷಿಸಿಲ್ಲ.
+
+ಪತ್ತರ್ಕೆ ಪನ್ನೆನಿಕ್ಕುಮ
+ದುತ್ತರಕೊಂದೆರಡು ನೆಲಸೆ ಪದಿಯೆಂದಕ್ಕುಂ
+ಮತ್ತೊದವಿ ಮೂಱು ನಾಲ್ಕಿರೆ
+ಸುತ್ತೇಂ ಸ್ವರಮಿರೆ ನಕಾರವಿಧಿ ಮುಂದೆಲ್ಲಂ ||[ಸೂತ್ರ 203](https://youtu.be/ZIjoq7hUcVc?list=PLYc_H2yt-vAJhMjmz3cztfjiNO7XhgRQG&t=3365)||
+
+ಪತ್ತು-ವಿಗೆ ವಿಭಕ್ತಿಸೂಚಕವಾದ ಅನುಸ್ವಾರ ಸಮೇತವಾದ ಪದಿಂ ಇತ್ಯಾದಿ ಆದೇಶವಾಗಲಾರದು [ಅನುಸ್ವಾರದ ಅನುಸಾರ ನೋಡಿ]. ವಿಭಕ್ತರೂಪದ ಅಂದರೆ ಪತ್ತಿಂ -ನ ಆದೇಶವಾಗಿರಬಹುದು ಅಷ್ಟೇ.
```

Comment:
```suggestion
ಪತ್ತು-ವಿಗೆ ವಿಭಕ್ತಿಸೂಚಕವಾದ ಅನುಸ್ವಾರ ಸಮೇತವಾದ `ಪದಿಂ` ಇತ್ಯಾದಿ ಆದೇಶವಾಗಲಾರದು [ಅನುಸ್ವಾರದ ಅನುಸಾರ ನೋಡಿ]. ವಿಭಕ್ತರೂಪದ ಅಂದರೆ `ಪತ್ತಿಂ` -ನ ಆದೇಶವಾಗಿರಬಹುದು ಅಷ್ಟೇ.
```

1. Hyperlink?
2. `ಪತ್` is the root, not `ಪತ್ತು`. Talk about the following alternative derivations because ಪತ್ತಿಂ ~ ಪದಿಂ needs further explanation?
  - `ಪತ್ + ಇಂ/ಇಁ`
    - `=> ಪತಿಂ/ಪತಿಁ` - without DaDum avoidance kicking in (plausible because this is from before Telugu, Tulu, Kannada, Tamil etc. split).
      - `=> ಪದಿಂ/ಪದಿಁ` - `ತ` turns into `ದ`.
    - `=> ಪತ್ತಿಂ/ಪತ್ತಿಁ` - with DaDum avoidance
      - `=> ಪದ್ದಿಂ/ಪದ್ದಿಁ` - `ತ` turns into `ದ`.

Reply:
modified to address these. please check.